lib/browser/api/browser-window.js

```diff
@@ -80,11 +80,51 @@ BrowserWindow.prototype._init = function () {
     set: (autoHide) => this.setAutoHideMenuBar(autoHide)
   })
 
+  Object.defineProperty(this, 'visibleOnAllWorkspaces', {
+    get: () => this.isVisibleOnAllWorkspaces(),
+    set: (visible) => this.setVisibleOnAllWorkspaces(visible)
+  })
+
+  Object.defineProperty(this, 'simpleFullScreen', {
+    get: () => this.isSimpleFullScreen(),
+    set: (simple) => this.setSimpleFullScreen(simple)
+  })
+
+  Object.defineProperty(this, 'kiosk', {
+    get: () => this.isKiosk(),
+    set: (kiosk) => this.setKiosk(kiosk)
+  })
+
+  Object.defineProperty(this, 'documentEdited', {
+    get: () => this.isFullscreen(),
+    set: (edited) => this.setDocumentEdited(edited)
+  })
+
+  Object.defineProperty(this, 'fullscreen', {
```

Comment:
Should be `fullScreen` to match the method name and `simpleFullScreen` property above.

Reply:
@zcbenz i did this intentionally since the constructor property is `fullscreen` with no capitalization and i felt it might confuse people 🤔

Reply:
Maybe leave this to discussion on next meeting? I don't have a strong opposition on this but I think we should be consistent with new APIs and fix the inconsistent old ones.

Reply:
Works for me! i agree that we should aim for consistency here.

Reply:
@codebytere Maybe leave `fullscreen` out in this PR, or use `fullScreen` temporarily? From the discussion in @electron/wg-api it seems that everyone agrees we should have a consistent API but the problem is to whether use `fullscreen` or `fullScreen`.

If we are going to use `fullscreen` then we have to deprecate a few APIs, so no whether we are doing it or not the change would be better put in a separate PR.